guides/micronaut-openapi-generator-server/java/src/main/java/example/micronaut/BookService.java

```diff
@@ -0,0 +1,33 @@
+package example.micronaut;
+
+import example.micronaut.model.BookAvailability;
+import example.micronaut.model.BookInfo;
+import jakarta.inject.Singleton;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Singleton // <1>
+public class BookService {
```

Comment:
why are we adding this file to the server guide?

Reply:
You are right. Missed it when merged with master.